src/components/connection-modal/connected-step.jsx

```diff
@@ -0,0 +1,57 @@
+import {FormattedMessage} from 'react-intl';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Box from '../box/box.jsx';
+import Dots from './dots.jsx';
+
+import styles from './connection-modal.css';
+import classNames from 'classnames';
+
+const ConnectedStep = props => (
+    <Box className={styles.body}>
+        <Box className={styles.activityArea} />
+        <Box className={styles.bottomArea}>
+            <Box className={styles.instructions}>
+                <FormattedMessage
+                    defaultMessage="Connected"
+                    description=""
+                    id="gui.connection.connected"
+                />
+            </Box>
+            <Dots
+                success
+                total={3}
+            />
+            <div className={styles.cornerButtons}>
+                <button
+                    className={classNames(styles.redButton, styles.connectionButton)}
+                    onClick={props.onDisconnect}
+                >
+                    <FormattedMessage
+                        defaultMessage="disconnect"
+                        description="Disconnect the device"
+                        id="gui.connection.disconnect"
+                    />
+                </button>
+                <button
+                    className={styles.connectionButton}
+                    onClick={props.onCancel}
+                >
+                    <FormattedMessage
+                        defaultMessage="go to editor"
+                        description=""
```

Comment:
These descriptions will be the context for translators to translate the text, so it should say something about the context.